api/transaction/routes.go

```diff
@@ -191,46 +192,39 @@ func GetTransaction(c *gin.Context) {
 
 	tx, err := ef.GetTransaction(txhash)
 	if err != nil {
-		c.JSON(http.StatusInternalServerError, gin.H{"error": errors.ErrGetTransaction.Error()})
+		c.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
 		return
 	}
 
 	if tx == nil {
-		c.JSON(http.StatusNotFound, gin.H{"error": errors.ErrTxNotFound.Error()})
```

Comment:
is possible method GetTransaction to return nil, nil ?  because i take a look in implementation and this method never returns nil, nil .... so this check should me removed.

Reply:
removed